optuna/pruners/hyperband.py

```diff
@@ -103,23 +144,18 @@ def __init__(
             )
             self._pruners.append(pruner)
 
-    def prune(self, study, trial):
-        # type: (optuna.study.Study, FrozenTrial) -> bool
-
+    def prune(self, study: Study, trial: FrozenTrial) -> bool:
         i = self._get_bracket_id(study, trial)
         _logger.debug("{}th bracket is selected".format(i))
         bracket_study = self._create_bracket_study(study, i)
         return self._pruners[i].prune(bracket_study, trial)
 
     # TODO(crcrpar): Improve resource computation/allocation algorithm.
-    def _calc_bracket_resource_budget(self, pruner_index, n_brackets):
-        # type: (int, int) -> int
-
+    def _calc_bracket_resource_budget(self, pruner_index: int, n_brackets: int) -> int:
         n = self._reduction_factor ** (n_brackets - 1)
         return n + (n / 2) * (n_brackets - 1 - pruner_index)
```

Comment:
[nits] I think that we can use `self._n_brackets` instead of the `n_brackets` argument.
```suggestion
    def _calc_bracket_resource_budget(self, pruner_index: int) -> int:
        n = self._reduction_factor ** (self._n_brackets - 1)
        return n + (n / 2) * (self._n_brackets - 1 - pruner_index)
```

Reply:
Thank you for your comment. This point is already fixed in PR #1141.